po/ja.po

```diff
@@ -0,0 +1,108 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.artemanufrij.imageburner package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.artemanufrij.imageburner\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-09-22 18:40+0200\n"
+"PO-Revision-Date: 2018-10-27 12:43+0100\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.2\n"
+"Last-Translator: Ciarán Ainsworth <ciaranainsworth@outlook.com>"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: ja\n"
+
+#: src/MainWindow.vala:67 src/MainWindow.vala:89
+msgid "Change"
+msgstr "変更"
+
+#: src/MainWindow.vala:70 src/MainWindow.vala:335
+msgid "Open"
+msgstr "開く"
+
+#: src/MainWindow.vala:97 src/MainWindow.vala:256
+msgid "Device"
+msgstr "デバイス"
+
+#: src/MainWindow.vala:124
+msgid "Calculating checksum…"
+msgstr "チェックサムの計算中…"
+
+#: src/MainWindow.vala:144
+msgid "please wait…"
+msgstr "お待ちください…"
+
+#: src/MainWindow.vala:156 src/MainWindow.vala:422
+msgid "Ready!"
+msgstr "準備完了!"
+
+#: src/MainWindow.vala:158
+#, c-format
+msgid "%s was written onto %s"
+msgstr "%s は %s に書き込まれました"
+
+#: src/MainWindow.vala:185
+msgid "Image Burner"
+msgstr "イメージバーナー"
+
+#: src/MainWindow.vala:194
+msgid "Choose an algorithm"
+msgstr "アルゴリズムを選択する"
+
+#: src/MainWindow.vala:211
+msgid "Finished"
+msgstr "完成した"
+
+#: src/MainWindow.vala:228
+msgid "Image"
+msgstr "イメージ"
+
+#: src/MainWindow.vala:236
+msgid "Select Image"
+msgstr "イメージを選択"
+
+#: src/MainWindow.vala:270
+msgid "Select Drive"
+msgstr "ドライブの選択"
+
+#: src/MainWindow.vala:279 src/MainWindow.vala:412
+msgid "No removable devices found…"
+msgstr "リムーバブルデバイスが見つかりませんでした…"
+
+#: src/MainWindow.vala:308
+msgid "Flash"
+msgstr "フラッシュ"
+
+#: src/MainWindow.vala:318
+msgid "Write Image"
+msgstr "イメージを書き込みます"
+
+#: src/MainWindow.vala:337
+msgid "_Cancel"
+msgstr "キャンセル(_C)"
+
+#: src/MainWindow.vala:338
+msgid "_Open"
+msgstr "開く(_O)"
+
+#: src/MainWindow.vala:341
+msgid "Image files"
+msgstr "イメージファイル"
+
+#: src/MainWindow.vala:404
+msgid "Choose an image file…"
+msgstr "イメージファイルを選択してください…"
+
+#: src/MainWindow.vala:418
+msgid "No image file chosen…"
+msgstr "イメージファイルファイルを選択していません…"
```

Comment:
Since original text is passive voice, we should reflect that voice to the translation. And "ファイル" is duplicated
```suggestion
msgstr "イメージファイルが選択されていません…"
```